integration_tests/mockApis/orchestration.ts

```diff
@@ -476,7 +476,7 @@ export default {
       },
     })
   },
-  stubGetFutureExcludeDates: ({
+  stubGetFutureBlockedDates: ({
     prisonId,
     prisonExcludeDates = [TestData.prisonExcludeDateDto()],
```

Comment:
Do these names match up any more? Future Blocked vs Excluded?

Reply:
Yeah - all pretty consistently 'block' (matching the actual content/design) in the front-end now rather than 'exclude' . API named differently 🙁